driver/main.c

```diff
@@ -1876,6 +1898,8 @@ TRACEPOINT_PROBE(syscall_exit_probe, struct pt_regs *regs, long ret)
 	}
 #endif
 
+	this_cpu_inc(g_n_tracepoint_hit);
```

Comment:
It seems like `this_cpu_inc` was introduced in 2.6.33 (https://github.com/torvalds/linux/commit/7340a0b15280c9d902c7dd0608b8e751b5a7c403), so it might be worth checking if it's at least included in EL6 (apologize if you already did). It probably is since they backport a bunch of things and at this point it's not worth anymore worrying about vanilla 2.6.32 (like Debian Squeeze)